media/css/firefox/nothing-personal/styles.scss

```diff
@@ -12,92 +12,232 @@ $image-path: '/media/protocol/img';
     font-family: 'Fira Mono', 'Andale Mono', monospace;
-    position: relative;
+    padding-top: $layout-lg;
+    padding-bottom: $layout-lg;
+    background: $page-background 60% 10% / cover no-repeat url('/media/img/firefox/nothing-personal/grid-background.svg');
+
+    @media (prefers-reduced-motion: no-preference) and (#{$mq-sm}) {
+        background-attachment: fixed;
+    }
+}
+
+.c-main {
+    font-weight: 500;
+
+    > *:not(:last-child) {
+        margin-bottom: $layout-xl; // IE fallback
+
+        @supports (--css: variables) {
+            --extra-spacing: #{$layout-xs};
+            margin-bottom: calc(#{$layout-xl} + var(--extra-spacing));
+
+            @media #{$mq-md} {
+                --extra-spacing: #{$layout-md};
+            }
+
+            // no stickers/gifs to fill space below
+            .c-privacy,
+            .c-features {
+                --extra-spacing: 0px;
+            }
+
+            // re-add spacing when fox gif displays
+            @media #{$mq-lg} {
+                .c-features {
+                    --extra-spacing: #{$layout-md};
+                }
+            }
+        }
+    }
 
-    section {
+    .mzp-t-content-md {
+        max-width: remify(510px);
+    }
+}
+
+.c-aside {
+    display: none;
+}
+
+@media #{$mq-lg} {
+    // non-grid layout, under main section
+    .c-aside {
+        display: flex;
+        justify-content: center;
+        margin-top: $spacing-xl;
+        text-align: center;
+    }
+
+    .c-aside-link {
+        color: $color-white;
         display: flex;
         flex-direction: column;
         align-items: center;
-        overflow-x: hidden;
+        padding: $spacing-sm;
+        text-decoration: none;
+        cursor: default;
+
+        + * {
+            margin-left: $spacing-md;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            color: $color-white;
+        }
+
+        &:active,
+        &:focus {
+            background-color: $color-blue-60;
+        }
     }
+}
+
+.c-text-section {
+    margin-bottom: $spacing-xl;
 
-    aside {
-        display: none;
+    * {
+        color: $color-white; // IE fallback
+        color: var(--body-text-color-inverse);
     }
+}
 
-    @media (prefers-reduced-motion: no-preference) and (#{$mq-sm}) {
-        background-attachment: fixed;
+@supports (display: grid) {
+    .c-grid {
+        display: grid;
+        gap: $spacing-md;
+
+        &.mzp-l-content::after {
+            display: none; // remove pseudo-element spacing
+        }
+
+        @media #{$mq-md} {
+            grid-template-columns: repeat(2, minmax(0, 1fr));
+
+            &.mzp-t-content-md {
+                max-width: $screen-md;
+            }
+        }
     }
 
+    // sidebar layout
     @media #{$mq-lg} {
-        aside {
-            display: flex;
+        .c-content-layout {
+            display: grid;
+            grid-template-columns: 1fr [main-start] auto [main-end aside-start] 1fr [aside-end];
```

Comment:
I thought this would help avoid some absolute positioning, but I've got to push the aside section over the main section padding anyway, so I'm on the fence about using grid over absolute positioning here. Reviewer's choice!